Lib/unittest/mock.py

```diff
@@ -807,7 +807,8 @@ def assert_called_once(_mock_self):
             raise AssertionError(msg)
 
     def assert_called_with(_mock_self, *args, **kwargs):
-        """assert that the mock was called with the specified arguments.
+        """assert that the last call to the mock was made with the specified
```

Comment:
The docstring's summary line should only be in one line. Any way this can be made more concise and less than 80 characters? 🤔

Reply:
Hi @Mariatta, I was not aware of this rule, thanks. What do you think of:

> assert that the last call was made with the specified arguments.

While the other docstrings need `the mock` to be grammatically correct, it seems to me that this one doesn't and the context here is sufficient?

Reply:
I think that would work.

Reply:
Thanks @csabella, I made the change.